### PR TITLE
Send support questions to the support ticket page

### PR DESCRIPTION
**Pull request by acasajus**
This MR

 - Will reply once to noreply@domain with a message redirecting users to the support form
 - Add a footer to all messages with a link to the support form

### CODE REVIEW

File: email_handler.py [hunk 1]
```diff
@@ -2078,6 +2079,20 @@ def should_ignore(mail_from: str, rcpt_tos: List[str]) -> bool:
     return False
 
 
+def send_no_reply_response(mail_from: str, msg: Message):
+    user = User.get_by(email=mail_from)
```
Comment (nguyenkims) [inline email_handler.py:2083]: It's possible that user replies to a noreply address from a mailbox which isn't his SL account address. I think we can maybe create a similar method to `send_email_at_most_times` which doesn't require the `user` parameter. This methods use `SentAlert` for rate limiting and SentAlert.user_id attribute is optional.

File: templates/emails/base.html [hunk 2]
```diff
@@ -482,7 +482,7 @@
                     {% block footer %}
                     {% endblock %}
                   </p>
-
+                  <p class="f-fallback sub align-center" style="font-size: 13px; line-height: 1.625; text-align: center; color: #A8AAAF; margin: .4em 0 1.1875em;" align="center">Do you have a question? Contact us at <a href="https://app.simplelogin.io/dashboard/support">https://app.simplelogin.io/dashboard/support</a></p>
```
Comment (nguyenkims) [inline templates/emails/base.html:485]: I think we can just keep "Contact us" in the href link. So something like `<a href="https://app.simplelogin.io/dashboard/support">A question?</a>`

File: templates/emails/base.txt.jinja2 [hunk 3]
```diff
@@ -0,0 +1,6 @@
+{% block content %} {% endblock %}
+
+Best,
+  SimpleLogin team.
```
Comment (nguyenkims) [inline templates/emails/base.txt.jinja2:4]: I think we can remove the space here, so something like:

```
Best,
SimpleLogin team.
```

File: templates/emails/transactional/noreply.text.jinja2 [hunk 4]
```diff
@@ -0,0 +1,7 @@
+Hi!
+
+    Thanks for getting in touch. This mailbox cannot receive any emails. If you need help, please go to https://app.simplelogin.io/dashboard/support to contact us.
```
Comment (nguyenkims) [inline templates/emails/transactional/noreply.text.jinja2:3]: Nits: I think we can remove some empty spaces at the beginning of some lines here

File: templates/emails/transactional/subscription-cancel.txt [hunk 5]
```diff
@@ -12,5 +12,3 @@ Best,
 Son.
 
 -----------------------------------------
-Son NK - SimpleLogin founder.
```
Comment (nguyenkims) [inline templates/emails/transactional/subscription-cancel.txt:15]: 👌

File: templates/emails/transactional/unsubscribe-newsletter.txt [hunk 6]
```diff
@@ -1,12 +1,9 @@
+{% extends "base.txt.jinja2" %}
+
+{% block content %}
 You have been unsubscribed from SimpleLogin newsletter.
 
 We would love to know why do you want to unsubscribe from our communication.
 
 Please let us know by replying to this email.
```
Comment (nguyenkims) [inline templates/emails/transactional/unsubscribe-newsletter.txt:8]: Nits: we can remove the "Please let us know by replying to this email." here

**Review by nguyenkims — COMMENTED** [review]
The PR looks good, thanks for going through all emails!

Just left some comments. We also need to change this line https://github.com/simple-login/app/blob/c415324932dd95f52c2292fe6abc8d08697b898b/app/email_utils.py#L299 to use the NOREPLY address instead of SUPPORT_EMAIL.

File: templates/emails/base.html (hunk 2 above)
Comment (acasajus) [reply templates/emails/base.html:485]: Changed.

File: templates/emails/base.txt.jinja2 (hunk 3 above)
Comment (acasajus) [reply templates/emails/base.txt.jinja2:4]: Changed

File: templates/emails/transactional/noreply.text.jinja2 (hunk 4 above)
Comment (acasajus) [reply templates/emails/transactional/noreply.text.jinja2:3]: Fixed

File: templates/emails/transactional/subscription-cancel.txt (hunk 5 above)
Comment (acasajus) [reply templates/emails/transactional/subscription-cancel.txt:15]: Oops. Too fast removing spaces to add the block templating. Restored.

File: templates/emails/transactional/unsubscribe-newsletter.txt (hunk 6 above)
Comment (acasajus) [reply templates/emails/transactional/unsubscribe-newsletter.txt:8]: Ooops. Removed

File: email_handler.py (hunk 1 above)
Comment (acasajus) [reply email_handler.py:2083]: How about trying to find out if there's a user with that email registered. And send it only once per user still. If we don't know the mail for any of our users it's probably safe to just skip it.

File: templates/emails/transactional/unsubscribe-newsletter.txt [hunk 7]
```diff
@@ -1,12 +1,7 @@
...
-
-Please let us know by replying to this email.
-
-Best,
-SimpleLogin Team.
-
-
-
-
+{% endblock %}
```
Comment (nguyenkims) [inline templates/emails/transactional/unsubscribe-newsletter.txt:7]: Nits: as user can't reply to this email anymore, we can remove the "We would love to know why do you want to unsubscribe from our communication." sentence.

File: email_handler.py [hunk 8]
```diff
@@ -2078,6 +2079,21 @@ def should_ignore(mail_from: str, rcpt_tos: List[str]) -> bool:
     return False
 
 
+def send_no_reply_response(mail_from: str, msg: Message):
+    mailbox = Mailbox.get_by(email=mail_from)
+    user = User.get(mailbox.user_id)
```
[the remark this answers is not in the document]
Comment (acasajus) [reply email_handler.py:2084]: Good catch. Fixed the bug.

File: templates/emails/transactional/unsubscribe-newsletter.txt (hunk 7 above)
Comment (acasajus) [reply templates/emails/transactional/unsubscribe-newsletter.txt:7]: Removed.

**Review by nguyenkims — APPROVED** [review]
LGTM!

**Pull request merged**